examples/deprecation/deprecation.out

```diff
@@ -0,0 +1,11 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file, and execute the `ballerina run` command.
+$ ballerina run deprecation.bal
+usage of construct 'Foo' is deprecated
```

Comment:
This is not how the warning is actually printed right? Isn't this prefixed with `warning:`?

Reply:
Done